packages/docs/content/foundation/typography/headings.mdx

```diff
@@ -68,60 +68,69 @@ Heading--4xl is used for the heading on landing pages that serve as an entry poi
 
 <ThemeContent neverThemes={['medicare']}>
 <EmbeddedExample>
-      <h1 class="ds-text-heading--4xl ds-title">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--4xl">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 </ThemeContent>
 
 ### Heading--3xl
 
 <EmbeddedExample>
-      <h1 class="ds-text-heading--3xl ds-h1">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--3xl">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 
 ### Heading--2xl
 
 <EmbeddedExample>
-      <h1 class="ds-text-heading--2xl ds-h2">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--2xl">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 
 ### Heading--xl
 
 <EmbeddedExample>
-      <h1 class="ds-text-heading--xl ds-h3">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--xl">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 
 ### Heading--lg
 
 <EmbeddedExample>
-      <h1 class="ds-text-heading--lg ds-h4">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--lg">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 
 ### Heading--md
 
 <EmbeddedExample>
-      <h1 class="ds-text-heading--md ds-h5">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--md">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 
 ### Heading--sm
 
 <EmbeddedExample>
-      <h1 class="ds-text-heading--sm ds-h6">{loremIpsumGenerator('s')}</h1>
+      <h1 class="ds-text-heading--sm">{loremIpsumGenerator('s')}</h1>
 </EmbeddedExample>
 
 ## Code
 
 ### Responsive typography
 
-The `.ds-type-heading-3xl`, `ds-type-heading-2xl`, and `ds-type-heading-xl` classes are responsive by default.
-To apply responsive typography elsewhere, use the [font size utility class](/utilities/font/font-size/#responsive-font-sizes).
-Since the base typography margins and line height is measured in `em` units,
-they'll automatically adjust as you change the font size.
+`.ds-type-heading-5xl`, `ds-type-heading-4xl`, and `ds-type-heading-3xl` classes are responsive by default.
+
+
+| Font class | 545px and below     | 545px to 768px | 768px and above                       |
+| ----------- | ------------ | ----- | -------------------------------------------------------------|
+|`.ds-type-heading-5xl`  | 36px    | 48px  | 60px    |
+|`.ds-type-heading-4xl`  | 36px    | 36px  | 48px    |
+|`.ds-type-heading-3xl` | 30px    | 30px  | 36px    |
```

Comment:
It would be nice to import these directly from our tokens package, but it would be out of scope for this ticket.

Reply:
I agree, maybe that can be the next update?